readme.md

```diff
@@ -124,5 +139,9 @@ Your Answer:
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
 ```text
+Fork creates an exact copy of any repo to my list of repositories.
+
+pull request incorporates changes from a remote repository into the current branch. In its default mode, git pull in shorthand for git fetch followed by git merge FETCH_HEAD.
+
```

Comment:
You have provided some decent definitions for fork, pull request, and pull.  But, this doesn't describe a git workflow for teams.